Fix bug in the default creation of VisMeta (#66)
TCDSolar/xrayvision: changelog/66.bugfix.rst

> Fix bug that happens when you create a Visibility object with default meta.

Reviewer: ```suggestion
Fix bug when you creating :class:`~xrayvision.visibility.Visibilities` with default meta.
```
